src/vehicle.cpp

```diff
@@ -1478,6 +1478,16 @@ bool vehicle::remove_part( int p )
         }
     }
 
+    //Remove loot zone if Cargo was removed.
+    const auto lz_iter = loot_zones.find( parts[p].mount );
+    const bool no_zone = lz_iter != loot_zones.end();
+
+    if( no_zone && part_flag( p, "CARGO" ) ) {
+        debugmsg( "cargo zone removed" );
```

Comment:
A vehicle losing a cargo part is a normal thing to happen, right?  In which case a `debugmsg` is probably not the correct thing to do.  You could put a message in the log with `g->u.add_msg_if_player`.

Reply:
My bad, I thought I finished removing all of those.